jest.config.base.js

```diff
@@ -0,0 +1,21 @@
+const { TextDecoder, TextEncoder } = require("util");
+
+module.exports = {
+  roots: ["<rootDir>/src"],
+  transform: {
+    "^.+\\.ts$": "ts-jest",
+  },
+  moduleFileExtensions: ["ts", "tsx", "js", "jsx", "json", "node"],
+  collectCoverage: true,
+  coveragePathIgnorePatterns: ["(tests/.*.mock).(jsx?|tsx?)$"],
```

Comment:
I can't find a test case which matches this regex, is this something we can omit? (Or maybe I'm misunderstanding where these files are generated?)

Reply:
Good catch